src/styles/components/_component-card.scss

```diff
@@ -1,20 +1,46 @@
-.component-link {
-  display: block;
-  outline: none;
+.component-card {
+  display: flex;
+  flex-direction: column-reverse;
+  height: 21.875rem; // 350px/16
   width: 100%;
-  height: 100%;
+  cursor: pointer;
+  transition: all 150ms ease-in-out;
+  border: 2px solid var(--foreground-color);
+  background-color: var(--background-color);
 
-  .component-card {
-    display: flex;
-    flex-direction: column-reverse;
-    height: 21.875rem; // 350px/16
-    width: 100%;
-    cursor: pointer;
+  &__image {
+    @media (min-width: $breakpoint-scale-up) {
+      height: 100%;
+    }
+  }
```

Comment:
If you delete line 11, and line 15 only, that's it! Otherwise the accordion is teeny!